### PR TITLE
Unit tests for displayAll method

### PR DESCRIPTION
**Pull request by MaxGranger97**

### CODE REVIEW

File: tests/ViewHelpers/PropertyViewHelperTest.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        $entity->expects($this->once())
+            ->method('getImage')
+            ->willReturn('Image');
+
+        $collection = $this->createMock(\ArmadilloEstates\Collections\PropertyCollection::class);
+        $collection->expects($this->once())
+            ->method('getAllProperties')
+            ->willReturn([$entity]);
+
+        $expected = '<div class="col-4 text-center mb-4"><div class="card"><h5 class="card-title pt-3">Address1 Address2<br>Town</h5><img class="card-img-top cardImg" src=Image alt="Card image cap"><div class="card-body"><p class="card-text">Sale</p><p class="card-text">Sold</p><a href="#" class="btn btn-primary">View More...</a></div></div></div>';
```
Comment (samcrudge) [inline tests/ViewHelpers/PropertyViewHelperTest.php]: Could you format this so it's more readable?

File: tests/ViewHelpers/PropertyViewHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        $this->assertEquals($expected, PropertyViewHelper::displayAll($collection));
+    }
+
+    public function testIncorrectEntityPassedDisplayAll()
+    {
+        $stdclass = $this->createMock(stdClass::class);
+
+        $this->expectException(TypeError::class);
+
+        PropertyViewHelper::displayAll($stdclass);
```
Comment (samcrudge) [inline tests/ViewHelpers/PropertyViewHelperTest.php]:
```suggestion
        PropertyViewHelper::displayAll($stdClass);
```

**Review by samcrudge — CHANGES_REQUESTED** [review]

File: tests/ViewHelpers/PropertyViewHelperTest.php (hunk 1 above)
Comment (turnerror) [reply tests/ViewHelpers/PropertyViewHelperTest.php]: no. the string needs to be on one line.
//edit 
im an idiot

File: tests/ViewHelpers/PropertyViewHelperTest.php [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        PropertyViewHelper::displayAll($stdClass);
+    }
+
+    public function testEmptyCollectionDisplayAll()
+    {
+        $collection = $this->createMock(\ArmadilloEstates\Collections\PropertyCollection::class);
+        $collection->expects($this->once())
+            ->method('getAllProperties')
+            ->willReturn([]);
+
```
Comment (samcrudge) [inline tests/ViewHelpers/PropertyViewHelperTest.php:65]: White space
```suggestion
```

**Review by samcrudge — CHANGES_REQUESTED** [review]

**Review by samcrudge — APPROVED** [review]

**Pull request merged**